src/views/Home.vue

```diff
@@ -2,59 +2,114 @@
+          :lng="station.position.lng"
+          :station-id="station.id"
+          :name-of-station="station.name"
+        ></map-marker>
+        <user-location
+          v-if="position.longitude != NaN"
+          :lat="position.latitude"
+          :lng="position.longitude"
+        >
+        </user-location>
       </div>
     </v-row>
+    <v-btn top right absolute @click="show = !show" style="z-index: 3" icon
+      ><v-icon x-large color="dark">mdi-bus-marker</v-icon>
+    </v-btn>
+    <buslines-card :show="show" />
   </v-container>
 </template>
 
 <script>
 import BusStopInfo from "../components/BusStopInfo";
+import MapMarker from "../components/MapMarker";
+import mapService from "../services/mapService";
+import UserLocation from "../components/UserLocation";
+import BuslinesCard from "../components/BuslinesCard";
+import { mapGetters } from "vuex";
+
+import { bus } from "../main.js";
+
 export default {
   name: "Home",
-  components: { BusStopInfo },
+  components: { BuslinesCard, UserLocation, MapMarker, BusStopInfo },
+
   data() {
     return {
       selected: [],
-      buslines: [
-        { number: 1, check: false },
-        { number: 2, check: false },
-        { number: 3, check: false },
-        { number: 4, check: false },
-        { number: 5, check: false },
-        { number: 6, check: false },
-        { number: 7, check: false },
-        { number: 8, check: false },
-        { number: 9, check: false },
-        { number: 20, check: false },
-        { number: 24, check: false },
-        { number: 55, check: false },
-        { number: 34, check: false }
-      ]
+      map: null,
+      dialog: false,
+      stationName: null,
+      stationId: null,
+      show: false,
+      markerCluster: null,
+
+      position: { longitude: NaN, latitude: NaN }
     };
+  },
+  computed: {
+    ...mapGetters(["stations"])
+  },
+  mounted() {
+    this.map = new window.google.maps.Map(this.$refs["map"], {
+      center: { lat: 51.202709, lng: 16.15371 },
+      zoom: 14,
+      disableDefaultUI: true,
+      styles: [
+        {
+          featureType: "poi",
+          elementType: "labels",
+          stylers: [{ visibility: "off" }]
+        }
+      ]
+    });
+    mapService.getBuslines().then(() => {});
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(
+        this.getPositionSuccess,
+        this.getPositionError
+      );
+    }
+  },
+
+  methods: {
+    getPositionSuccess(pos) {
+      this.position = pos.coords;
+    },
+    getPositionError(err) {
+      console.warn(`ERROR(${err.code}): ${err.message}`);
+    },
+    getMap(callback) {
+      let vm = this;
+      function checkForMap() {
+        if (vm.map) callback(vm.map);
+        else setTimeout(checkForMap, 200);
+      }
+
+      checkForMap();
+    }
+  },
+  created() {
+    bus.$on("openEvent", (stationName, stationId) => {
+      if (this.dialog === true) this.dialog = !this.dialog;
```

Comment:
this if block is not needed here. You are doing this same stuff 3 lines bellow

Reply:
If statement is used in the situation when i want to open dialog when my dialog is already opened right?
To clarify, when i want to move from one to another station when checking timetable of busline.